ta/mk/build-user-ta.mk

```diff
@@ -2,6 +2,13 @@ include mk/cleanvars.mk
 sm := $(lastword $(subst /, ,$(dir $(ta-mk-file))))
 sm-$(sm) := y
 
+ifeq ($(ta-target),ta_arm32)
```

Comment:
Will this work? `ta-target` is reassigned below.

Reply:
Indeed this works. Maybe not for the good reasons :)

Reply:
How about replacing with with
```
ifeq ($(sm),ta_arm32)
arch-bits-ta_arm32 := 32
endif
ifeq ($(sm),ta_arm64)
arch-bits-ta_arm64 := 64
endif
```

below the assignment to be more clear and do it the same way as in `ta/mk/ta_dev_kit.mk`